lib/item.rb

```diff
@@ -41,9 +41,29 @@ def units_sold
     end
   end
 
+  def best_day
+    days_and_revenues.max_by { |k, v| v }[0]
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.